src/common/request/request.ts

```diff
@@ -14,14 +14,14 @@ export interface ServerResponse {
   errors?: string[]
 }
 
-class ResponseError extends Error {
-  status: number
-  errors?: ServerResponse['errors']
-
-  constructor(message: string, status: number, errors?: ServerResponse['errors']) {
+export class ResponseError extends Error {
+  public readonly name = 'ResponseError'
+  constructor(
+    message: string,
+    public readonly status: number,
+    public readonly errors?: ServerResponse['errors'],
+  ) {
     super(message)
-    this.status = status
-    this.errors = errors
```

Comment:
I did this because I was having issues and was trying to figure out why. This can be reverted if you want.

Reply:
Yes please revert if there's no issue

Reply:
reverted